src/box/lua/net_box.lua

```diff
@@ -1067,6 +1067,12 @@ local function new_sm(host, port, opts, connection, greeting)
     if opts.wait_connected ~= false then
         remote._transport.wait_state('active', tonumber(opts.wait_connected))
     end
+    -- Last stream ID used for this connection
+    -- == 0 - no streams was created
+    -- >0   - a number of streams was created with autoincrement ID
+    -- <0   - a number of streams was created with explicitly specified ID
+    remote._last_stream_id = 0
```

Comment:
Because we need to have access to these variables from the remote_methods:stream()

Reply:
But if it would be global, you still would be a be able to get it, wouldn't you?

Reply:
But if it will be global it will be same for all connections! But in different connections it should be different! It may be same stream id for different streams in different connections! Moreover for one connection user can use autoincrement stream_id and for other connection he can set it manually.